selfdrive/camerad/cameras/camera_qcom.cc

```diff
@@ -177,12 +177,12 @@ static int imx298_apply_exposure(CameraState *s, int gain, int integ_lines, int
     {0x3002,0x0,0}, // long autoexposure off
 
     // FRM_LENGTH
-    {0x340, frame_length >> 8, 0}, {0x341, frame_length & 0xff, 0},
+    {0x340, (unsigned short)(frame_length >> 8), 0}, {0x341, (unsigned short)(frame_length & 0xff), 0},
```

Comment:
Include stdint and do uint16_t instead of unsigned short?

Reply:
ok